apps/meteor/client/views/room/components/body/composer/messageBox/MessageBoxReplies.tsx

```diff
@@ -21,30 +23,44 @@ const MessageBoxReplies = (): ReactElement | null => {
 		return null;
 	}
 
+	const closeWrapperStyle = css`
+		position: absolute;
+		right: 0.5rem;
+		top: 0.75rem;
+	`;
+
 	return (
-		<div className='reply-preview__wrap message-popup'>
-			{replies.map((reply) => (
-				<div className='reply-preview message-popup'>
-					<div className='message'>
-						<Attachments attachments={[{ text: reply.msg, author_name: reply.u.username } as any]} />
-					</div>
-					<div
-						className='rc-message-box__icon cancel-reply'
-						data-mid='{{replyMessageData._id}}'
-						onClick={(): void => {
-							chat.composer?.dismissQuotedMessage(reply._id);
-						}}
-					>
-						<svg
-							className={`rc-icon rc-message-box__toolbar-formatting-icon rc-message-box__toolbar-formatting-icon--cross`}
-							aria-hidden='true'
-						>
-							<use xlinkHref={`#icon-cross`} />
-						</svg>
-					</div>
-				</div>
-			))}
-		</div>
+		<Box mbe='x8' position='relative' overflowY='auto' maxHeight='x256'>
+			{replies.map((reply, key) => {
+				console.log(reply);
```

Comment:
we dont need a `console.log` here. dont you agree?

Reply:
😢